tower-web/src/styles.scss

```diff
@@ -26,42 +26,74 @@ code {
 }
 
 #navbar {
-  width: 100%;
-  height: $navbar-height;
-  background-color: white;
   box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.16);
-  padding-top: 20px;
-  padding-bottom: 20px;
+  width: 100vw;
 }
 
 .navbar-brand img {
-  width: 260px;
+  height: 40px;
 }
 
 #layout {
-  display: flex;
-  min-height: calc(100% - #{$navbar-height});
   padding-top: $navbar-height;
-  width: 100%;
+}
+
+.sidebar-toggler-btn {
+  color: rgba(0,0,0,.5);
+  cursor: pointer;
+  padding: .25rem .75rem;
+  font-size: 1.25rem;
+  line-height: 1;
+  background-color: transparent;
+  border: 1px solid rgba(0,0,0,.1);
+  border-radius: .25rem;
+}
+
+#sidebar-wrapper {
+  min-height: 100vh;
+  margin-left: $sidebar-width * -1;
+  -webkit-transition: margin .25s ease-in-out;
+  -moz-transition: margin .25s ease-in-out;
+  -o-transition: margin .25s ease-in-out;
+  transition: margin .25s ease-in-out;
+  width: $sidebar-width;
+}
+
+#sidebar-wrapper.toggled {
+  margin-left: 0;
 }
 
 #sidebar {
-  position: sticky;
-  top: 0;
-  flex: 0 0 25%;
+  // TODO: Make sticky-top behaviour work with navbar height
```

Comment:
Currently this hides underneath the top-nav when you scroll, I need to fix that still.